package.json

```
@@ -30,6 +30,7 @@
     "is-string-a-number": "2.0.2",
     "lodash": "4.17.11",
     "modularscale-sass": "3.0.8",
+    "polished": "^3.2.0",
```

On peut fixer la version ?

---

Oui, il y a besoin de faire quoi, ça ira comme ça ?

```suggestion
    "polished": "3.2.0",
```